src/get-started/install.md

```diff
@@ -19,6 +19,7 @@ to these projects, testing them, and developing using Exonum.
 Exonum depends on the following third-party system libraries:
 
 - [LevelDB][leveldb] (persistent storage)
+- [RocksDB][rocksdb] (persistent storage)
```

Comment:
Could you please clarify: is LevelDB and RocksDB optional or necessary dependencies? It seems that both are optional judging by [Cargo.toml](https://github.com/exonum/exonum/blob/master/exonum/Cargo.toml) in the `exonum` crate.

Reply:
@slowli `LevelDB` is default feature https://github.com/exonum/exonum/blob/master/exonum/Cargo.toml#L54

Reply:
Thanks! But it's still possible to compile and use Exonum without it, right? If so, I'd advise adding something like this below the list of dependencies:

> `LevelDB` and `RocksDB` are used as alternative storage engines, with `LevelDB` used by default. You need to install only the storage engine(s) you are intending to use.

Reply:
Yes, you are right.